python/triton/language/core.py

```diff
@@ -360,13 +360,21 @@ def to_ir(self, builder: ir.builder) -> ir.type:
     def __str__(self):
         return self.name
 
+    def codegen_name(self):
```

Comment:
Could you add a comment explaining why repr should be different from str?

Reply:
Added to repr function